HttpRequest.cpp: In constructor 'HttpRequest::HttpRequest(FILE*)':
HttpRequest.cpp:74: error: 'environ' was not declared in this scope

Undefined symbols:
  "_environ", referenced from:
      HttpRequest::HttpRequest(__sFILE*)in HttpRequest.o
      HttpRequest::HttpRequest(__sFILE*)in HttpRequest.o
ld: symbol(s) not found

http://wiki.finkproject.org/index.php/Fink:Porting_Notes#The__environ_symbol
--- HttpRequest.cpp.orig	2011-01-22 03:19:51.000000000 -0600
+++ HttpRequest.cpp	2011-03-29 01:32:20.000000000 -0500
@@ -63,6 +63,12 @@
 
 // --------------------------------------------------------------------------------------
 #ifndef _WIN32
+#ifdef __APPLE__
+#include <crt_externs.h>
+#define environ (*_NSGetEnviron())
+#else
+extern char **environ;
+#endif
 HttpRequest::HttpRequest(FILE *fil) : HttpTransaction()
 , m_server_port(0)
 , m_is_ssl(false)
